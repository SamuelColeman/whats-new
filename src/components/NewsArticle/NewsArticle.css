.newsArticle {
	border: 1px solid black;
	width: 50%;
	background: gray;
}

img {
	width: 100%;
	height: 50%;
}