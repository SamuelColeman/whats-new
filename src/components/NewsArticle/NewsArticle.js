import React from 'react';
import './NewsArticle.css';

const NewsArticle = (props) => {
	return (
		<div>
			<p>{props.img}</p>
			<h2>{props.headline}</h2>
			<p>{props.description}</p>
			<p>{props.url}</p>
		</div>
	)
}

export default NewsArticle;