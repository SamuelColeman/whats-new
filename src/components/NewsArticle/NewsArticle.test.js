import React from 'react';
import { shallow } from 'enzyme';
import NewsArticle from './NewsArticle';

describe('NewsArticle', () => {
	it('should match the snapshot', () => {
		const wrapper = shallow(<NewsArticle
			img='https://upload.wikimedia.org/wikipedia/commons/f/f0/Mops_oct09_cropped2.jpg'
			headline='Pugs Take Over'
			description='Pugsly wins presidential race.'
			url='https://en.wikipedia.org/wiki/Pug'
			/>)
		expect(wrapper).toMatchSnapshot();
	})
})