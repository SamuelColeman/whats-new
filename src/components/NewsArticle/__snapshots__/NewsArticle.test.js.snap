// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsArticle should match the snapshot 1`] = `
<div
  className="newsArticle"
>
  <img
    src="https://upload.wikimedia.org/wikipedia/commons/f/f0/Mops_oct09_cropped2.jpg"
  />
  <h2>
    Pugs Take Over
  </h2>
  <p>
    Pugsly wins presidential race.
  </p>
  <a
    href="https://en.wikipedia.org/wiki/Pug"
  >
    Full Article
  </a>
</div>
`;
